srcs/config/ConfigParse.cc

```diff
@@ -42,7 +42,7 @@ ConfigParse::ConfigParse(const Lines& file_data) : lines_(file_data), map_(), lo
 
   this->loc_map_ = {{"index", &ConfigParse::indexParse},       {"autoindex", &ConfigParse::autoIndexParse},
                     {"root", &ConfigParse::rootParse},         {"allowed_methods", &ConfigParse::allowedMethodsParse},
-                    {"redirect", &ConfigParse::redirectParse}, {"upload_dir", &ConfigParse::uploadDirParse}};
+                    {"redirect", &ConfigParse::redirectParse}, {"cgi_upload_dir", &ConfigParse::uploadCgiDirParse}};
```

Comment:
it feels kind of weird to me to call this directory cgi_upload_dir when cgi does not handle the uploads

Reply:
Oh this was a remnant from before